<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>654</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>654</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CD/DVD Management Software</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Path/Drive to Index</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Group Name/ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CD/DVD Name/ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Database</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dvdid">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="groupid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="comments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="size_morethan">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="filetype">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fullpath">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="allgroups">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Include all DVD/CD Groups</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DVD/CD Group:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> File Type/EXT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullpath</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating &quot;&gt;&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Comments</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Size &quot;&gt;&quot;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="size_lessthan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Size &quot;&lt;&quot;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dvdgroups_list">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="searchresults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>581</width>
      <height>251</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>GroupID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DVDID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Filename</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ext</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fullpath</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rating</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Comments</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="ratings">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionCreate"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionIndex_Files"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
   <property name="toolTip">
    <string>Create Database</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Commit/Save Database</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load an existing Database</string>
   </property>
  </action>
  <action name="actionIndex_Files">
   <property name="text">
    <string>Index Files</string>
   </property>
   <property name="toolTip">
    <string>Index Files after choosing the path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
